src/Value/Message.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+declare(strict_types=1);
+
+namespace KafkaRest\Value;
+
+use JMS\Serializer\Annotation as Serializer;
+
+final class Message
+{
+    /**
+     * @Serializer\Type("string")
+     * @Serializer\SkipWhenEmpty()
```

Comment:
```suggestion
     * @Serializer\SkipWhenEmpty
```

can be done everywhere

Reply:
Sure can, but I thought we keep the brackets on purpose ... same as when instantiating objects.